root/static/scripts/release-editor/fields.js

```diff
@@ -387,6 +398,11 @@ class Track {
     return capitalizedETIRegex.test(this.name());
   }
 
+  hasMiscapitalizedTitle() {
+    return this.medium.release.shouldUseEnglishCapitalization() &&
```

Comment:
I'm wondering if I should try to find some way to also exclude releases and/or tracks credited to artists with Japanese areas, since this may annoy people who edit a lot of English-language Japanese releases otherwise. It might be a bit tricky, since it looked to me like artist areas aren't currently being passed to the release editor (I didn't dig deeper to see if they're being loaded from the DB, but I suspect they may not be).

On the other hand, this change just displays the yellow warning box without the checkbox for non-beginners, so maybe it's not a big deal. `¯\_(ツ)_/¯`

Reply:
I'm fairly sure they will survive. They already get an exception, they can live with ignoring a yellow box.